src/main/java/task/Task.java

```diff
@@ -0,0 +1,30 @@
+package task;
+
+public class Task {
+    public String description;
+    public boolean isDone;
+
+    public Task(String description){
+        this.description = description;
+        this.isDone = false;
+        System.out.println("Got it. I've added this task:");
+    }
+
+    public String toString(){
+        String output = "[ ] ";
+        if(isDone){
+            output = "[X] ";
+        }
+
+        return output + this.description;
+    }
+
+    public void setDone(boolean status){
```

Comment:
Booleans should be made to sound like booleans as per the Java Coding standard. Consider renaming `status` to `isDone` as well.